### PR TITLE
Add details about fees in trade API

### PR DESCRIPTION
**Pull request by dnfd**

### CODE REVIEW

**Comment by mod** [comment]
I would show only maker_fee amount and currency

File: app/api/v2/admin/entities/trade.rb [hunk 1]
```diff
@@ -40,6 +40,54 @@ class Trade < API::V2::Entities::Trade
...
+            }
+          ) { |trade| trade.taker_order.taker_fee }
+
+          expose(
+            :maker_fee_amount,
+            documentation: {
+              type: BigDecimal,
+              desc: 'Trade maker fee amount.',
+            }
+          ) { |trade| trade.maker_order.maker_fee * trade.maker_order.funds_received }
```
Comment (ysv) [inline app/api/v2/admin/entities/trade.rb]: it's wrong see here how I fetch it https://github.com/rubykube/peatio/pull/2337/files#diff-7a3e5fe6af9dc29d3988d79ced7b0cd3

File: app/api/v2/admin/entities/trade.rb [hunk 2]
```diff
@@ -40,6 +40,54 @@ class Trade < API::V2::Entities::Trade
...
+
+          expose(
+            :taker_fee_amount,
+            documentation: {
+              type: BigDecimal,
+              desc: 'Trade taker fee amount.',
+            }
+          ) { |trade| trade.taker_order.taker_fee * trade.taker_order.funds_received }
```
Comment (ysv) [inline app/api/v2/admin/entities/trade.rb]: it's wrong see here how I fetch it https://github.com/rubykube/peatio/pull/2337/files#diff-7a3e5fe6af9dc29d3988d79ced7b0cd3

**Review by ysv — CHANGES_REQUESTED** [review]

**Review by ysv — CHANGES_REQUESTED** [review]
Add tests

File: app/api/v2/entities/trade.rb [hunk 3]
```diff
@@ -37,6 +37,17 @@ class Trade < Base
           }
         )
 
+        expose(
```
Comment (ysv) [inline app/api/v2/entities/trade.rb:40]: Also add fee & fee currency

**Review by ysv — APPROVED** [review]

**Pull request merged**